conda_smithy/configure_feedstock.py

```diff
@@ -478,12 +478,12 @@ def update_matrix(forge_config, new_matrix):
 
 def prepare_matrix_for_env_vars(matrix):
     """
-    Turns a matrix with environment variables and pakages into a matrix of
+    Turns a matrix with environment variables and packages into a matrix of
     just environment variables. The package variables are prefixed with CONDA,
-    and special cases such as Python and Numpy are handled.
+    and special cases such as Python, NumPy, and R are handled.
 
     """
-    special_conda_vars = {'python': 'CONDA_PY', 'numpy': 'CONDA_NPY'}
+    special_conda_vars = {'python': 'CONDA_PY', 'numpy': 'CONDA_NPY', 'r-base': 'CONDA_R'}
```

Comment:
It's interesting that we don't do this with `perl`, yet we still get a matrix with it. So am unclear if this is needed at this time. Though may need to look at this more closely.